src/platform/packages/shared/kbn-scout-upgrade-testing/test/discovery_tests/discovery.spec.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { expect, test } from '@kbn/scout';
+
+const defaultSettings = {
+  defaultIndex: 'logstash-*',
+  'dateFormat:tz': 'UTC',
+};
+
+test.describe('discover test', { tag: ['@ess'] }, () => {
+  test.beforeAll(async ({ kbnClient, esArchiver }) => {
+    await kbnClient.importExport.load(
+      'src/platform/test/functional/fixtures/kbn_archiver/discover'
+    );
+    await esArchiver.loadIfNeeded(
+      'src/platform/test/functional/fixtures/es_archiver/logstash_functional'
+    );
+    await kbnClient.uiSettings.update(defaultSettings);
+  });
+
+  test.beforeEach(async ({ browserAuth, pageObjects }) => {
+    await browserAuth.loginAsAdmin();
+    await pageObjects.discover.goto();
+  });
+
+  test.afterAll(async ({ kbnClient }) => {
+    await kbnClient.savedObjects.clean({ types: ['search', 'index-pattern'] });
+  });
+
+  test.describe('timepicker queries', () => {
+    const queryName1 = 'Query # 1';
+    const queryName2 = 'Query # 2';
+
+    test('should show correct time range string by timepicker', async ({ pageObjects }) => {
+      await pageObjects.timePicker.setDefaultAbsoluteRange();
```

Comment:
question: do we test the date picker functionality here or just that data id correct?